src/collections/_documentation/development/sdk-dev/event-payloads/breadcrumbs.md

```diff
@@ -3,14 +3,16 @@ title: Breadcrumbs Interface
 sidebar_order: 6
 ---
 
-The breadcrumbs interface specifies a series of application events, or
-`breadcrumbs`, that occurred before the main event. Its canonical name is
-`breadcrumbs`.
+The Breadcrumbs Interface specifies a series of application events, or
+"breadcrumbs", that occurred before and led up to an event.
 
-This interface is an object with a single `values` key containing an ordered
-list of breadcrumb objects. The entries are ordered from oldest to newest.
-Consequently, the last entry in the array should be the last entry before the
-event occurred.
+An [event]({%- link _documentation/development/sdk-dev/event-payloads/index.md
+-%}) may contain one or more breadcrumbs in an attribute named `breadcrumbs`.
+
+This interface is an object with a single `values` attribute containing an
+ordered list of breadcrumb objects. The entries are ordered from oldest to
+newest. Consequently, the last entry in the list should be the last entry before
+the event occurred.
 
 Each breadcrumb has a few properties of which at least `timestamp` and
```

Comment:
```suggestion
Each breadcrumb has a few properties you must provide, such as `timestamp` and
```

Reply:
@rhcarvalho Yeah, on line 18 I deleted "must be provided." Does it not show up?

Reply:
It should read "Each breadcrumb has a few properties you mush provide such as `timestamp` and `category`."

Reply:
> Does it not show up?

I missed it right under my nose 🤦‍♂ 

I'll have to check the implementation because here we say `timestamp` and `category` are required, but then in the [list of attributes](https://docs.sentry.io/development/sdk-dev/event-payloads/breadcrumbs/#attributes) we say `category` is `optional`.

Reply:
@jan-auer or @untitaker could you please help clarify what is actually required for a _breadcrumb_?

Part of the docs says `timestamp` and `category`, part says only `timestamp`. In practice, nothing seems to be strictly required.

I looked at the code and sent a sample event. In practice, it works to send an empty breadcrumb `{"breadcrumbs": {"values": [{}]}`, which the server [normalizes](https://github.com/getsentry/relay/blob/5069fbc7846782f372bfc0dee522e0aae7d9b673/relay-general/src/store/normalize.rs#L474-L491) as `{"breadcrumbs":{"values":[{"type":"default","level":"info"}]}}`.

Reply:
This is how an "empty" breadcrumb like described in my previous comment renders:

![image](https://user-images.githubusercontent.com/88819/78020204-4b3c2700-7351-11ea-8d18-968a76835a26.png)

The breadcrumb is in the "generic" line.

Reply:
After talking with Markus, decided to update the docs to reflect the truth, i.e., nothing is strictly required. We can change the wording from **required** to **recommended**, as otherwise the value from the breadcrumb is rather limited, and in some cases the non-timestamped information could be sent in a different field (e.g. a tag).

Reply:
Since this was not part of the original diff, I'll make changes to what is required/recommended in a new PR.

Reply:
Sent https://github.com/getsentry/sentry-docs/pull/1579 to clarify what is required/recommended/optional.